setup.sh

```diff
@@ -0,0 +1,113 @@
+if [[ -z "${INTEGRATION_PORTAL_URL}" ]]; then
+    echo "INTEGRATION_PORTAL_URL must be provided"
+    exit
+fi
+
+if [[ -z "${LEAPP_ROLES}" ]]; then
+    echo "LEAPP_ROLES must be provided"
+    exit
+fi
+
+# Install the xcode Command Line Tools, if they are not installed
+xcode-select --install
+
+# Install Homebrew if not installed
+which -s brew
+if [[ $? != 0 ]] ; then
```

Comment:
This is a great feature. It would be awesome to have a similar script for the New Engineer Setup!